tests/dev_wrappers/test_torch_wrapper.py

```diff
@@ -94,3 +95,27 @@ def test_torch_wrapper_vector_env():
 
     (obs, *_) = envs.step(torch.Tensor(envs.action_space.sample()))
     assert all(isinstance(o, torch.Tensor) for o in obs)
+
+
+@pytest.mark.parametrize(
+    "value",
+    [
+        1.0,
+        2,
+        (3.0, 4.0, 5.0),
+        [3.0, 4.0, 5.0],
+        {
+            "a": 6.0,
+            "b": 7,
+        },
+    ],
+)
+def test_torch_conversion(value):
+    assert torch_to_jax(jax_to_torch(value)) == value
```

Comment:
I added the tests prior to making changes to handle `Tuple`s and the tests just passed. I believe `Tuple`s are being caught under the `abc.Mapping` conversion.

Reply:
Yeah agreed